### DISCUSSION TITLE
Unofficial or official?

### DISCUSSION THREAD

**Discussion by cgomesu** [doitandbedone/ispyagentdvr-docker#366] [Q&A] [answered]
Why is this repo and the dockerhub images (https://hub.docker.com/r/doitandbedone/ispyagentdvr) being referred to as **unofficial** when they are actually referred to in the Docker installation instructions on the ispyconnect website itself?

![image](https://user-images.githubusercontent.com/50534116/153249584-1bad5a4c-f998-46f9-9ba5-70b25f544136.png)

Is it because it is not under @ispysoftware?  I can see they've contributed to this repo though.  I could not find any info about this in issues and discussions and I'm a bit confused about the use of the unofficial label.

**Comment by doitandbedone** [comment, marked as answer]
Right, I created the image to contribute to the community as there is a
need for it. This is not directly developed or maintained by ispy thus not
calling it unofficial. I get support from them from time to time as needed
for it to be running, however.

You are welcomed to use the image and is openly available with its source.
You may also use the natively supported platforms from their website.

I'll probably add this to the FAQs.

**Reply by cgomesu** [reply]
that's alright.  thanks for the prompt reply and for working on this Docker implementation.  I've other unrelated questions that I will ask separately once I'm done checking what has been done so far.  marking your reply as the answer to my original question.
